ui/components/MeshModelRegistry/MeshModelComponent.js

```diff
@@ -222,7 +221,12 @@ const MeshModelComponent_ = ({
       const tempResourcesDetail = [];
 
       for (let registrant of registrants) {
-        let hostname = toLower(registrant?.hostname);
+        let hostname = registrant?.hostname;
+
+        // Check if the hostname is "Artifacthub" and change it to "Artifact Hub"
+        if (hostname === 'Artifacthub') {
```

Comment:
Thanks. We need to change this at source in the Meshery Integrations spreadsheet.

If the spreadsheet has the correct spelling, then we need to make sure that any processing done in Golang is not changing the name.

Reply:
Hey @leecalcote, thank you for your review. Could you please share the link to the Meshery Integrations spreadsheet? I'll promptly verify the correct spelling in the spreadsheet and conduct a thorough review of our Golang code to ensure that it doesn't unintentionally alter the names.

Reply:
Its in the community drive.

Reply:
I've checked the Meshery Integrations spreadsheet and confirmed the correct spelling of the name. However, upon reviewing our Golang code, I couldn't find any instances where the name is being altered unintentionally.